utils/cta_dbt_helper.sh

```diff
@@ -0,0 +1,199 @@
+        if [[ $? != 0 ]]; then
+            echo "Ctrl-C caught, exiting..."
+            exit 1
+        fi
+    done
+    TARGET_PATH="$ROOT_PATH/dbt-cta/$DIR_NAME/"
+    gum confirm "Confirm the Target Path is correct: $TARGET_PATH" || exit 1
+    # Check to see if Directory exists, if not create it
+    if [[ ! -d $TARGET_PATH ]]; then
+        mkdir -p $TARGET_PATH
+    fi
+
+    # Get path to exported Airbyte Workspace
+    while [ -z "$WORKSPACE" ]; do
+        WORKSPACE=$(gum input --prompt "Enter the path of the exported Airbyte Workspace: " --placeholder "airbyte_dbt_export/814/")
+        if [[ $? != 0 ]]; then
+            echo "Ctrl-C caught, exiting..."
+            exit 1
+        fi
+        WORKSPACE_PATH="$ROOT_PATH/$WORKSPACE"
+    done
+    WORKSPACE_PATH="$ROOT_PATH/$WORKSPACE"
+    gum confirm "Confirm the Airbyte Worspace Path is correct: $WORKSPACE_PATH" || exit 1
+    # Check to see if Workspace Path actually exists
+    if [[ ! -d $WORKSPACE_PATH ]]; then
+        echo "The entered path for the Airbyte Workspace doesnt exist. Path: $WORKSPACE_PATH"
+    fi
+    pipenv run python $ROOT_PATH/utils/dbt_format_utils.py restructAirbyteDbt --airbyteWorkspacePath $WORKSPACE_PATH --dbtCtaPath $TARGET_PATH
+
+}
+
+append_base_to_files() {
+    # Get target path for directory that should be modified from user
+    while [ -z "$DIR_NAME" ]; do
+        DIR_NAME=$(gum input --prompt "Enter the directory name of the base tables to rename: " --placeholder "(ex. dbt-cta/actblue/1_cta_incremental/)")
+        if [[ $? != 0 ]]; then
+            echo "Ctrl-C caught, exiting..."
+            exit 1
+        fi
+    done
+    gum confirm "Confirm the Base models path is correct: $ROOT_PATH/$DIR_NAME" || exit 1
+    echo "Starting job to rename base model files..."
+    echo ""
+    pipenv run python $ROOT_PATH/utils/dbt_format_utils.py renameBaseTables --basePath $ROOT_PATH/$DIR_NAME
+    echo ""
+    echo "Base models have been renamed! You can copy/pasta the outputted list into your sources.yml file."
+}
+
+generate_matviews() {
+    # Get target path for directory holding Base models
+    while [ -z "$INPUT_DIR_NAME" ]; do
+        INPUT_DIR_NAME=$(gum input --prompt "Enter the directory name of the base models to generate matviews for: " --placeholder "(ex. dbt-cta/actblue/1_cta_incremental/)")
+        if [[ $? != 0 ]]; then
+            echo "Ctrl-C caught, exiting..."
+            exit 1
+        fi
+    done
+    gum confirm "Confirm the Base models path is correct: $ROOT_PATH/$INPUT_DIR_NAME" || exit 1
+
+    # Get target path for directory where matviews should be created
+    while [ -z "$OUTPUT_DIR_NAME" ]; do
+        OUTPUT_DIR_NAME=$(gum input --prompt "Enter the directory name where matviews should be placed: " --placeholder "(ex. dbt-cta/actblue/2_partner_matviews/)")
```

Comment:
@jitoquinto can we revise the placeholder here so it includes the `models` subdirectory? 🙏 

```suggestion
        OUTPUT_DIR_NAME=$(gum input --prompt "Enter the directory name where matviews should be placed: " --placeholder "(ex. dbt-cta/actblue/models/2_partner_matviews/)")
```